### ISSUE TITLE
getReact as a child command should throw a good error if subject is empty

### ISSUE THREAD

**Issue by bahmutov** [abhinaba-ghosh/cypress-react-selector#25]
if I accidentally query a non-existent component

```js
cy.getReact('ProductContainer') // does not exist
  .getReact('AProduct')
```

Then the error thrown is super cryptic

<img width="973" alt="Screen Shot 2020-08-11 at 10 36 21 PM" src="https://user-images.githubusercontent.com/2212006/89969043-7568f400-dc23-11ea-9114-a62685902592.png">

It would be nice to 

- throw a good error if the subject is empty
- auto-retry finding the component

**Comment by bahmutov** [comment]
For now I do insert
```js
cy.getReact('ProductContainer')
  .should('not.be.empty')
  .getReact('AProduct')
```

**Comment by abhinaba-ghosh** [comment]
Hi @bahmutov , Let me know your thought in the below error message. If looks good, I gonna merge the changes. 

<img width="573" alt="Screenshot 2020-08-12 at 1 26 37 PM" src="https://user-images.githubusercontent.com/23141842/89990177-9a9b4800-dc9f-11ea-8fb3-0f2a929ce0fe.png">

**Comment by bahmutov** [comment]
Looks much better 👍

**Comment by abhinaba-ghosh** [comment]
fixed in v2.0.1
